wolfssl/test.h

```diff
@@ -834,6 +834,7 @@ static INLINE void tcp_accept(SOCKET_T* sockfd, SOCKET_T* clientfd,
 {
     SOCKADDR_IN_T client;
     socklen_t client_len = sizeof(client);
+    tcp_ready* ready = NULL;
 
```

Comment:
It looks like there is a scenario where "ready" is not used. Either wrap the "ready" declaration in the right combination of #if defined logic or do a "(void)ready" to suppress the un-used arg warning.